src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -255,6 +255,11 @@ export const evidenceTypeHelp = (
 );
 
 const capitalizeEach = (word) => {
+  // Prevent calling toUpperCase() on undefined because of a double space in the name
+  if (!word) {
```

Comment:
Another option might be to do `split(/ +/)`

Reply:
Oh hey. Forgot that `split` can take a regex too. I like that better.